src/TLabel.h

```diff
@@ -39,10 +41,40 @@ Q_OBJECT
 
 public:
 
-                  TLabel( QWidget * pW=0 );
-void              setScript( Host * pHost, QString & func, TEvent * args ){ mpHost = pHost; mScript = func; mpParameters = args; }
-void              setEnter( Host * pHost, QString & func, TEvent * args ){ mpHost = pHost; mEnter = func; mEnterParams = args; }
-void              setLeave( Host * pHost, QString & func, TEvent * args ){ mpHost = pHost; mLeave = func; mLeaveParams = args; }
```

Comment:
Yeah, ___that___ makes sense as passing by value __is__ safe to do even if we'd do otherwise in other circumstances. :relieved: